curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/step-025.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-25
 
 # --description--
 
-Now update your `h1` selector to be `header h1`, to specifically target your `h1` element within your new `header`.
+Now update your `h1` selector to be `header h1` to specifically target your `h1` element within your new `header`.
```

Comment:
I am not sure why we removed this comma. This sentence consists of two clauses.

Reply:
It's always dangerous when I pretend to know grammar, but I don't think you need to have a comma between the main and subordinate clauses.

Reply:
Sorry, I should have waited with merging for more insights from those not scared of comma grammar.

Reply:
It is not a big deal; the sentence is relatively easy to follow. So, my nit-pickiness aside, this is fine.